.gitignore

```diff
@@ -39,8 +39,9 @@ avmmakefile
 README-generated.md
 avm.tflint.hcl
 avm.tflint_example.hcl
+avm.tflint.merged.hcl
+avm.tflint_example.merged.hcl
 *tfplan*
 *.md.tmp
-
```

Comment:
This disturbs me, leave my space please :D

Reply:
that's what grept-apply said to do, i have no animosity towards blank lines 😂